### DISCUSSION TITLE
Increase width of tab group icons?

### DISCUSSION THREAD

**Discussion by rieje** [mbnuqw/sidebery#1617] [Q&A]
Is it possible to increase the width of [tab group icons](https://0x0.st/XHHA.png)? I would like to give it a text value but it's too small. Also would it be possible to have tab group icons expand multi-line so they remain visible (as opposed to hidden behind a menu or requiring horizontal scrolling) when there's more than a few of them?

**Comment by madeddy** [comment]
I have basically the same questions. The ability to make the tab-panel icons bigger, would be of great value to me. 
Any way to do this?

**Comment by lmBored** [comment]
Those are the child elements of Navigation Bar:
```css
.NavigationBar
```

You can modify `.nav-item`'s width and height to increase the size of tab group icons.

Reference: [Navigation bar source code](https://github.com/mbnuqw/sidebery/blob/v5/src/styles/themes/proton/sidebar/bar.navigation.styl)

**Reply by madeddy** [reply]
Thanks for taking the time.

> You can modify `.nav-item`'s width and height to increase the size of tab group icons.

> Another option is to go to Settings > Styles editor > Navigation > Button width

Both are IMO the same and they do not change the **ICON** of the button. The button grows but the icon inside stays the same size:

Original state:
![sideberry_navbutton1](https://github.com/user-attachments/assets/70b88ef8-108d-4d76-984a-41b9255aefd8)

Individual state:
![sideberry_navbutton2](https://github.com/user-attachments/assets/f936c12f-1b3c-456e-8c1d-7871efecd505)

I want the icon scalable or the icon scaling automatically relative to his button size.

**Reply by lmBored** [reply]
Oh for that, you can simply do:
```css
.nav-item .icon {
	width: 50px !important;
	height: 50px !important;
}
```

**Reply by madeddy** [reply]
> Oh for that, you can simply do:

This works. Finally!

Thanks a lot! 😃

**Reply by lmBored** [reply]
no problem!

**Comment by lmBored** [comment]
Another option is to go to `Settings > Styles editor > Navigation > Button width` and modify the width to your liking.
